unison-src/transcripts/redundant.md

```diff
@@ -0,0 +1,15 @@
+The same kind of thing happens with `map`. Are we saying this is incorrect behaviour?
```

Comment:
@runarorama was this file accidentally committed? Or if not maybe flesh out this file with some more verbiage.

Reply:
I have no idea where this came from. Merge from master?

Reply:
Here's git blame: https://github.com/unisonweb/unison/blame/ef640acb3ba5fdc91fc36c3a859763c9215b6151/unison-src/transcripts/redundant.md